src/lightning/fabric/utilities/seed.py

```diff
@@ -123,6 +131,9 @@ def _set_rng_states(rng_state_dict: Dict[str, Any]) -> None:
     # torch.cuda rng_state is only included since v1.8.
     if "torch.cuda" in rng_state_dict:
         torch.cuda.set_rng_state_all(rng_state_dict["torch.cuda"])
+    if "torch.xpu" in rng_state_dict:
+        if XPUAccelerator.is_available():
+            XPUAccelerator._set_rng_states(rng_state_dict)
```

Comment:
better to use `torch.xpu`, rather than `XPUAccelerator`.

Reply:
Same as above. if we write torch.xpu here again, then we could have done for all methods. I thought this too hence I wrote in the lightning-xpu class.